bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+		<label>PM10</label>
+		<description>The Particulate Matter 10 (0.01mm).</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="co2">
+		<item-type>Number:Dimensionless</item-type>
+		<label>CO2</label>
+		<description>The measured CarbonDioxide.</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="filterLife">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Filter Life</label>
+		<description>The filter life indication.</description>
+		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="doorOpen">
+		<item-type>Contact</item-type>
+		<label>Door Status</label>
+		<description>The door status Open/Closed.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fanSpeed">
+		<item-type>Number</item-type>
+		<label>Fan Speed Setting</label>
+		<description>The fan speed setting.</description>
+		<state>
+			<options>
+				<option value="1">Level 1</option>
+				<option value="2">Level 2</option>
+				<option value="3">Level 3</option>
+				<option value="4">Level 4</option>
+				<option value="5">Level 5</option>
+				<option value="6">Level 6</option>
+				<option value="7">Level 7</option>
+				<option value="8">Level 8</option>
+				<option value="9">Level 9</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="workMode">
+		<item-type>String</item-type>
+		<label>Work Mode Setting</label>
+		<description>The work mode setting.</description>
+		<state>
+			<options>
+				<option value="PowerOff">Power Off</option>
+				<option value="Auto">Automatic</option>
+				<option value="Manual">Manual</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="ionizer">
+		<item-type>Switch</item-type>
+		<label>Ionizer Status</label>
+		<description>The ionizer status.</description>
+	</channel-type>
+
+	<channel-type id="uiLight">
+		<item-type>Switch</item-type>
+		<label>UI Light</label>
+		<description>The air quality light status.</description>
+	</channel-type>
+
+	<channel-type id="safetyLock">
+		<item-type>Switch</item-type>
+		<label>Safety Lock</label>
+		<description>The safety lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorLock">
+		<item-type>Switch</item-type>
+		<label>Door Lock</label>
+		<description>The door lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorState">
+		<item-type>Contact</item-type>
+		<label>Door Status</label>
+		<description>The door status Open/Closed.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="connectionState">
+		<item-type>Switch</item-type>
+		<label>Connection State</label>
+		<description>The connection state status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="startTime">
+		<item-type>Number:Time</item-type>
```

Comment:
If i udnerstand it correct, this channel holds a timestamp. If so, the item type should be `DateTime`

Reply:
Need to check this, I think it holds seconds since all other times is seconds.

Reply:
If it holds seconds it is more like a duration, i find that rather confusing for a startTime.
Does it holds seconds from epoch to indicate the start time, in that case i suggest to use the `TimeZoneProvider` to create an actual datetime as that is more usuable /readable.

Reply:
I have now tested, this is the json returned:
```
{
  "applianceId": "XXXXXXXXXXXXXXXXXXXX",
  "connectionState": "connected",
  "status": "enabled",
  "properties": {
    "reported": {
      "displayLight": "DISPLAY_LIGHT_0",
      "doorState": "CLOSED",
      "timeToEnd": 1200,
      "miscellaneous": {
        "defaultSoftPlus": false
      },
      "applianceUiSwVersion": "UDC0M40Y",
      "applianceTotalWorkingTime": 1483200,
      "remoteControl": "ENABLED",
      "language": "ENGLISH",
      "fCMiscellaneousState": {
        "optisenseResult": 65535,
        "detergentExtradosage": 40,
        "tankAReserve": true,
        "tankBReserve": true,
        "softenerExtradosage": 20,
        "waterUsage": 0,
        "tankADetLoadForNominalWeight": 0,
        "tankBDetLoadForNominalWeight": 0
      },
      "cyclePhase": "UNAVAILABLE",
      "endOfCycleSound": "SHORT_SOUND",
        "startTime": 7200,
```

So when you use the delayed start function, you can choose to delay the start in whole hours from 1-20.
So a startTime of 7200, means that the washing will start in 7200/60/60=2h from now.

During a delayed start, timeToEnd then indicates the washing program time.
After pressing start, the startTime will count down to 0, then the prgram will start and then timeToEnd will count down to 0.

What is the best way of representing this?
`Number:Time` as they are right now, or `DateTime`?

If we would use DateTime then startTime would be:
`ZonedDateTime.now(timeZoneProvider.getTimeZone()).plusSeconds(startTime);`
and timeToEnd would then be:
`ZonedDateTime.now(timeZoneProvider.getTimeZone()).plusSeconds(startTime + timeToEnd);`

Reply:
I found this PR by @jlaur:
https://github.com/openhab/openhab-addons/pull/13841/files

From this it maybe leans towards using Number:Time.

Reply:
In that case it is a duration and yes then a `Number:Time` is the best. Maybe it can also be modelled as a zoneddatetime, but that is a little error prone. As then you have to combine a duration with the actual datetime and the duration is from a remote service, so it will always be off for a couple of seconds. 

Maybe better to leave the current handling as is and only rename the channel. I find `startTime` confusing, maybe `time-to-start` is the best as it also nicely alligns with the `time-to-end` channel.

This would also make it possible to (in the future) create a second channel that represent the same value as actual datetimee might that ever be needed. Without breaking compatibility.

Reply:
Sounds like a plan.